db/src/main/java/io/lantern/db/DB.kt

```diff
@@ -993,32 +1040,14 @@ class Transaction internal constructor(
     /**
      * Deletes the value at the given path
      */
-    fun <T : Any> delete(path: String, extractFullText: ((T) -> String)? = null) {
+    fun delete(path: String) {
         val serializedPath = serde.serialize(path)
-        extractFullText?.let {
-            db.rawQuery(
-                "SELECT rowid, value FROM ${schema}_data WHERE path = ?", arrayOf(serializedPath)
-            ).use { cursor ->
-                if (cursor != null && cursor.moveToNext()) {
-                    db.execSQL(
-                        "INSERT INTO ${schema}_fts(${schema}_fts, rowid, value) VALUES('delete', ?, ?)",
```

Comment:
Since we're using a normal fts5 table now, we don't need this special delete syntax anymore.